.github/workflows/pandoc.yml

```diff
@@ -0,0 +1,27 @@
+name: Create README.pdf
+
+on: push
```

Comment:
maybe not on push?
It will give several commit or push rebase step

Reply:
Which would you suggest then, `pull_request`? It should be somewhat similar to push but you can only check the PDF with a PR opened.

Reply:
Looks like I need to keep it like it is, somehow when using `pull_request` I don't have access to a normal git repo, or it is in a weird state?

Reply:
I guess pull_request only trigger the action when creating pr.
Maybe using label or comment `create-pdf!` to trigger this action.
It's not important currently, so it is also fine if the commits always have the creation only in last one.

Reply:
Now it should push only when updating `README.md`, which should make this more bearable.

Reply:
It's nice!